account_invoice_constraint_chronology/model/account_move.py

```diff
@@ -34,13 +34,18 @@ def _get_older_conflicting_invoices_domain(self):
             ]
         )
 
-    def _raise_older_conflicting_invoices(self):
+    def _raise_older_conflicting_invoices(self, older_move):
         self.ensure_one()
         raise UserError(
             _(
-                "Chronology conflict: A conflicting draft invoice dated before "
-                "{date_invoice} exists, please validate it first."
-            ).format(date_invoice=format_date(self.env, self.invoice_date))
+                "Chronology conflict: A conflicting draft invoice {name} for "
+                "{partner} dated before {date_invoice} exists, please validate it "
+                "first or remove its invoice date."
+            ).format(
```

Comment:
You should use translation function arguments instead of `format()`

Reply:
Shouldn't the translation functions be automatically validated with precommit? I thought that if the precommit was passing, it was because it was a valid format (in my personal experience, I get an error if I don't use the translation function).

Reply:
Yes, but this is new from >= v16. So, maybe you can propose a change in pylint-odoo.